app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ActiveRecord::Base
+
+  validates :name, :email, presence: true
+  validates :name, :email, uniqueness: true
```

Comment:
You can do this lines in one declaration:
`validates :name, :email, presence: true, uniqueness: true`

Reply:
What do yo think about validating email format?

Reply:
Thanks Elle, I like the idea of validating the email format! I have added it to my project todo list!